docs/shared/core-features/run-tasks.md

```diff
@@ -92,6 +88,45 @@ Note that Nx parallelizes all these tasks making also sure they are run in the r
 
 Learn more about the [run-many](/nx-api/nx/documents/run-many) command.
 
+### Run Tasks in Parallel
```

Comment:
Yeah I know what you mean. I mostly moved this out of the [Pipeline Configuration](https://nx.dev/concepts/task-pipeline-configuration#run-tasks-in-parallel) because I don't even know why it ended up there. So it made sense to integrate here as an additional thing to show. But I"m with you, this is something that could be split out. It won't hurt for now I think though